lib/cloud-operation/cloud-operation-v1.ts

```diff
@@ -79,13 +89,20 @@ class CloudOperationV1 extends CloudOperationBase implements ICloudOperation {
 
 	private pollForLogs(): void {
 		this.logsCheckInterval = setInterval(async () => {
-			await this.getCloudOperationLogs();
+			if (this.snoozeLogPoll) {
+				return;
+			}
 
 			const status = this.serverStatus.status;
-			if (status === CloudOperationBase.OPERATION_COMPLETE_STATUS || status === CloudOperationBase.OPERATION_FAILED_STATUS) {
+			const hasCompleted = status === CloudOperationBase.OPERATION_COMPLETE_STATUS || status === CloudOperationBase.OPERATION_FAILED_STATUS;
```

Comment:
You can change this to:

```suggestion
			const hasCompleted = this.serverStatus.status !== CloudOperationBase.OPERATION_IN_PROGRESS_STATUS;
```

Reply:
This is a personal preference, but I would rather check for the completed status values - e.g. if we add another non-completed status, we will have to add it here. But, if you would prefer this, I will change it.